src/rules/at-use-no-redundant-alias/README.md

```diff
@@ -0,0 +1,38 @@
+# at-use-no-redundant-alias
+
+By default, the module's namespace is just the last component of the module’s URL.
```

Comment:
Please check the format for the rule description from the developer guide:
https://stylelint.io/developer-guide/rules#write-the-readme

Reply:
Thanks !!